README.md

```diff
@@ -8,130 +8,47 @@ See also [badgr-ui](https://github.com/concentricsky/badgr-ui), the Angular fron
 ### About the Badgr Project
 Badgr was developed by [Concentric Sky](https://concentricsky.com), starting in 2015 to serve as an open source reference implementation of the Open Badges Specification. It provides functionality to issue portable, verifiable Open Badges as well as to allow users to manage badges they have been awarded by any issuer that uses this open data standard. Since 2015, Badgr has grown to be used by hundreds of educational institutions and other people and organizations worldwide. See [Project Homepage](https://badgr.org) for more details about contributing to and integrating with Badgr.
 
-## How to get started on your local development environment.
+## How to get started on your local development environment (Linux)
+Visit https://github.com/concentricsky/badgr-server for README.md
+
+## How to get started on your local development environment (Windows)
 Prerequisites:
```

Comment:
According to this PyCharm is not a prerequisite for the rest of the installation, however, PyCharm is being used later on in the README